OpenEXR/IlmImf/ImfZipCompressor.cpp

```diff
@@ -60,7 +60,9 @@ ZipCompressor::ZipCompressor
     _outBuffer (0),
     _zip(maxScanLineSize, numScanLines)
 {
-    _outBuffer = new char[_zip.maxCompressedSize()];
+    // TODO: Remove this when we can change the ABI
+    (void) _maxScanLineSize;
```

Comment:
why the (void) _maxScanLineSize?

Reply:
same as the other case. Hence the TODO - when we decide to break the vtables of things next, this member should be removed as it is unused.

Reply:
created an issue to clean these up eventually.